### PR TITLE
Admin_Pages

### PR DESCRIPTION
**Pull request by minhln-1284**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51804)

## WHAT (optional)
- I added the admin bootstrap theme to our project, and added i18n to it

## HOW
- I created 2 different layouts using namespace and structure views/controller and css files accordingly

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by minhln-1284** [comment]
Ready

File: app/views/admin/static_pages/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,157 @@
+
+  <nav class="sb-topnav navbar navbar-expand navbar-dark bg-dark">
+    <!-- Navbar Brand-->
+    <%= link_to "Admin Ecommerce" , admin_root_path, :class=> "navbar-brand ps-3"%>
```
Comment (ghost) [inline app/views/admin/static_pages/index.html.erb]:
```suggestion
    <%= link_to "Admin Ecommerce" , admin_root_path, :class=> "navbar-brand ps-3" %>
```

File: app/views/admin/static_pages/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,157 @@
+
```
Comment (ghost) [inline app/views/admin/static_pages/index.html.erb:7]: bỏ dòng trông

File: app/views/admin/static_pages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      <ul class="navbar-nav ms-auto ms-md-0 me-3 me-lg-4">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" id="navbarDropdown" href="#" role="button" data-bs-toggle="dropdown"
+            aria-expanded="false"><i class="fas fa-user fa-fw"></i></a>
+          <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
+            <li><a class="dropdown-item" href="#!"><%= t "admin.static_pages.settings" %></a></li>
+            <li>
+              <hr class="dropdown-divider" />
+            </li>
+            <li><a class="dropdown-item" href="#!"><%= t "admin.static_pages.log_out" %></a></li>
```
Comment (ghost) [inline app/views/admin/static_pages/index.html.erb]: Dùng link_to

File: app/views/layouts/application_admin.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Site Admin</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'application_admin', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application_admin', 'data-turbolinks-track': 'reload' %>
```
Comment (ghost) [inline app/views/layouts/application_admin.html.erb]: Dùng "

File: config/database.yml.example [hunk 5]
```diff
@@ -1,19 +1,13 @@
 default: &default
   adapter: mysql2
-  encoding: utf8
+  encoding: utf8mb4
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
-  password: 123
-  host: 127.0.0.1
-  port: 3306
+  password: Hashira123_X
```
Comment (ghost) [inline config/database.yml.example]: Sao lại đưa cả password lên thế này em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by minhln-1284** [comment]
Ready

**Pull request merged**